modules/apps/frontend-css/frontend-css-web/src/main/resources/META-INF/resources/taglib/_management_bar.scss

```diff
@@ -7,6 +7,21 @@
 	}
 }
 
+.management-bar-v2 .subnav-tbar {
+	.label .label-item .lexicon-icon-times {
+		height: 0.875em;
+		width: 0.875em;
```

Comment:
Instead of doing this, does `.label-item .lexicon-icon-times { padding: 0.15625rem; }` work without resetting the `font-size` on `.label .close` and `.label-lg .close`?